lib/game.rb

```diff
@@ -1,15 +1,24 @@
 class Game
-  attr_reader :console, :welcome_message, :display
+  attr_reader :console, :welcome_message, :display, :prompt, :board_manager
 
-  def initialize(console, welcome_message, display)
+  # attr_accessor :latest_move
+
+  def initialize(console, welcome_message, display, prompt, board_manager)
```

Comment:
`Game` is starting to have quite a few collaborators. I would suggest you start to think about if there are additional responsibilities within the `Game` that can be extracted. A large number of dependencies can be an indication of multiple responsibilities.

Reply:
I don't think you should refactor now necessarily, but something to think about as we tackle future stories.